Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -75,11 +75,12 @@ public async Task<ContestDetailsServiceModel> GetContestDetails(int id)
         {
             var user = this.userProviderService.GetCurrentUser();
             var contest = await this.contestsData.GetByIdWithProblems(id);
+            var isLecturerOrAdmin = contest != null && this.lecturersInContestsBusiness.IsUserAdminOrLecturerInContest(contest);
 
             var validationResult = this.contestDetailsValidationService.GetValidationResult((
                 contest,
                 id,
-                user) !);
+                isLecturerOrAdmin) !);
```

Comment:
Do you need the `!`?

Reply:
Making the Contest parameter nullable in GetValidationResult makes it possible to remove it